src/lib/components/Footer/FooterCopyright.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { FC, PropsWithChildren } from 'react';
+
+export type CopyrightProps = PropsWithChildren<{
+  href?: string;
+  by?: string | undefined;
```

Comment:
Redundant

Reply:
What is redundant @tulup-conner?

Reply:
`?` implies the `| undefined`

Reply:
Totally make sense, I believe and the effect is the same for adding the "any". Right?